canonical-web-and-design/vanilla-framework
# Remove deprecated icon styles

## Done

Remove last batch of deprecated styling (icons).

Fixes #3744 

## QA

- Open [demo](https://vanilla-framework-3960.demos.haus/docs/patterns/icons)
- Make sure all deprecated icons are removed from the docs page and the code
- Drive-by: make sure all icons are correctly displayed in cards in the docs (next to their label)
